K8s-deployment/Charts/auditing-server/README.md

```diff
@@ -0,0 +1,441 @@
+| ------------------------ | --------------------------------------------------------------------------------------- | --------------- |         |
+| `kubeVersion`            | Override Kubernetes version                                                             | `""`            |
+| `nameOverride`           | String to partially override common.names.fullname                                      | `""`            |
+| `fullnameOverride`       | String to fully override common.names.fullname                                          | `""`            |
+| `commonLabels`           | Labels to add to all deployed objects                                                   | `{}`            |
+| `commonAnnotations`      | Annotations to add to all deployed objects                                              | `{}`            |
+| `clusterDomain`          | Kubernetes cluster domain name                                                          | `cluster.local` |
+| `extraDeploy`            | Array of extra objects to deploy with the release                                       | `[]`            |
+| `diagnosticMode.enabled` | Enable diagnostic mode (all probes will be disabled and the command will be overridden) | `false`         |
+| `diagnosticMode.command` | Command to override all containers in the deployment                                    | `["sleep"]`     |
+| `diagnosticMode.args`    | Args to override all containers in the deployment                                       | `["infinity"]`  |
+
+
+### Image Parameters
+
+| Name                        | Description                                | Value                |
+| --------------------------- | ------------------------------------------ | -------------------- |
+| `image.registry`            | image registry                             | `ghcr.io`            |
+| `image.repository`          | image repository                           | `datakaveri/auditing-server-depl` |
+| `image.tag`                 | image tag (immutable tags are recommended) | `3.0-2a93bdf`        |
+| `image.pullPolicy`          | image pull policy                          | `IfNotPresent`       |
+| `image.pullSecrets`         | image pull secrets                         | `{}`                 |
+| `image.debug`               | Enable image debug mode                    | `false`              |
+| `containerPorts.http`       | HTTP container port                        | `8080`                 |
+| `containerPorts.https`      | HTTPS container port                       | `8443`                |
+| `containerPorts.hazelcast`  | Hazelcast container port                   | `5701`               |
+| `containerPorts.prometheus` | Prometheus container port                  | `9000`               |
+| `podAnnotations`            | Annotations for pods                       | `nil`                |
+
+
+### ApiServer Parameters
```

Comment:
Correctly add deployment parameters needed for auditing server. 
Api server,database,authenticator verticles , service and ingress does not exist 
in auditing server